sips/sip-029/sip-029-halving-alignment.md

```diff
@@ -0,0 +1,152 @@
+
+
+| Coinbase Reward Reduction Phase | Bitcoin Block Height | Approximate Date (time b/w halvings) | STX Reward (reduction) | STX Supply (after) |
+|--------------------------------|---------------------|-------------------------------------|----------------------|-------------------|
+| Current                        | -                   | -                                   | 1000                 | -                 |
+| 1st*                     | 945,000             | ~April 2026 (+1.33 yrs)            | 500 (50%)           | 1,607,907,038     |
+| 2nd                     | 1,050,000           | ~April 2028 (2 yrs)                | 250 (50%)           | 1,652,242,188     |
+| 3rd                      | 1,260,000           | ~April 2032 (4 yrs)                | 125 (50%)           | 1,696,547,013     |
+| 4th                      | 1,470,000           | ~April 2036 (4 yrs)                | 100 (20%)           | 1,718,699,425     |
+| -                          | -                   | ~Jan 2050 (13.833 yrs)             | 100 (0%)             | 1,779,628,415     |
+
+
+With these changes, miner incentives and PoX yield remain unchanged for another 16 months, which we believe is sufficient time for the nascent Nakamoto signer set to develop into a set of stable, professional block signers, and for the sBTC project to attract sufficient initial liquidity.
+
+# Related Work
+
+While many blockchain protocols implement token emission schedules with periodic reductions in block rewards, we are not aware of any precedent for modifying an emission schedule specifically to maintain economic incentives during major protocol upgrades.
+
+# Backwards Compatibility
+
+This change requires a hard fork of the Stacks blockchain
+
+# Activation
+
+## Voting Timeline
+
+Voting will begin at bitcoin block height 870,750, which occurs ~ Sunday, November 17th, 2024.
+Voting will conclude at bitcoin block height 872,750, which occurs ~ Sunday, December 1st, 2024.
+
+## Activation
+
+The SIP-029 STX emission schedule is designed to activate on Stacks 3.0 as defined in [SIP-021](https://github.com/stacksgov/sips/blob/feat/sip-021-nakamoto/sips/sip-021/sip-021-nakamoto.md). Therefore, this SIP is only meaningful when SIP-021 activates.
+
+### Process of Activation
+
+Users can vote to approve this SIP with either their locked/stacked STX or with unlocked/liquid STX, or both. The SIP voting page can be found at [stx.eco](https://stx.eco). The criteria for the stacker and non-stacker voting is as follows.
+
+#### For Stackers:
+
+In order for this SIP to activate, the following criteria must be met by the set of Stacked STX:
+
+- At least 80 million Stacked STX must vote, with at least 80% (64 million) voting "yes".
```

Comment:
What if 100 million STX voted "no"?

Reply:
then the sip isn't approved. 
here, the floor requirement is that 80 million stacked stx *must* vote. if there is more, even better. of those that vote (assuming the threshold of 80 million is reached), at least 80% must vote yes. 

if 100 million vote no in your scenario, then it would take another 400 million voting yes (at a minimum)